src/pages/settings/schedules/common/components/EmailStatement.tsx

```diff
@@ -170,7 +170,6 @@ export function EmailStatement(props: Props) {
           withoutAction
           clearInputAfterSelection
           exclude={schedule.parameters.clients}
-          staleTime={Infinity}
```

Comment:
@beganovich @turbo124 Setting staleTime to Infinity manually for the Schedule pages is no longer necessary because we have already changed it to be set by default.